instrumentation/net/http/otelhttp/transport.go

```diff
@@ -77,7 +77,11 @@ func (t *Transport) applyConfig(c *config) {
 }
 
 func defaultTransportFormatter(_ string, r *http.Request) string {
-	return "HTTP " + r.Method
+	path := r.URL.Path
```

Comment:
I don't think we can use the path (e.g., `/users/1234`) here.  The default HTTP handler instrumentation doesn't have route information (e.g., `/users/{id}`) available to it and so I think we need to use just the method.

> Instrumentation MUST NOT default to using URI path as span name, but MAY provide hooks to allow custom logic to override the default span name.

Reply:
Ah, 🤦 That makes sense. So I can drop the path part and just keep the change that removes HTTP.

Reply:
Looking at this some more... I guess that's what the `WithRouteTag` is available for in the handler. Checking out [this example](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/a08e95c39ab3858bfba02ace6ea0a21a989a5cf3/instrumentation/net/http/otelhttp/handler_example_test.go#L59), this satisfies the "MAY provide hooks to allow custom logic to override the default span name". 

```go
...
	otelhttp.WithRouteTag("/hello/:name", http.HandlerFunc(
		func(w http.ResponseWriter, r *http.Request) {
			ctx := r.Context()
...
```